### DISCUSSION TITLE
Can we change the file format of genotype probabilities file?

### DISCUSSION THREAD

**Discussion by XingerTang** [AlphaGenes/AlphaPeel#116] [General] [answered]
@gregorgorjanc @AprilYUZhang @AudreyAAMartin

It is the only file that the `AlphaPeel` that puts empty lines between the data of each individual. 

Even though the empty lines are not in the documentation, they are indeed in the [output files](https://github.com/AlphaGenes/AlphaPeel/blob/ed89acfe01aeaea2ea8f4f4f00d6fcea3d07d692/tests/functional_tests/test_rec/true-haps.txt).

**Comment by gregorgorjanc** [comment, marked as answer]
@XingerTang yeah, I was not aware of this. It's definitely not in line with other output so let's indeed make this file format consistent with the rest. Thanks for bringing this up! Looking forward to a PR linked to this discussion/issue (will turn it into issue)
